eunkyung/models/userDao.js

```diff
@@ -0,0 +1,30 @@
+const { mysqlDataSource } = require("./datasource");
+
+const createUser = async (name, password, email, profileImage) => {
+  return await mysqlDataSource.query(
+    `INSERT INTO users (
+      name, 
+      password, 
+      email,
+      profile_image
+      ) 
+    VALUES (?, ?, ?, ?);`,
+    [name, password, email, profileImage]
+  );
+};
+
+const emailCheckLogin = async (email) => {
```

Comment:
해당 함수는 email 데이터로 유저의 정보를 조회하는 것이기 때문에 함수명을 다음과 같이 수정해주세요!
```suggestion
const getUserByEmail = async (email) => {
```